From 62b7ebee85557452900dd59aea27f1ebe46e5838 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 9 Jan 2019 11:55:44 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-evb-rev3-android-lp4: Add
 vop-bw-dmc-freq

This patch also enables dmc auto freq.

Change-Id: Ic9de96c38190490dee2bdbdf86a2d8e8844ba9d5
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts      | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
index 5b370eaaed17..e1d09948c8f1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-lp4.dts
@@ -125,8 +125,13 @@
 		SYS_STATUS_DUALVIEW     800000
 		SYS_STATUS_ISP          800000
 	>;
+	vop-bw-dmc-freq = <
+	/* min_bw(MB/s) max_bw(MB/s) freq(KHz) */
+		0       577      400000
+		578     99999    800000
+	>;
 	auto-min-freq = <400000>;
-	auto-freq-en = <0>;
+	auto-freq-en = <1>;
 };
 
 &dmc_opp_table {
-- 
2.35.3

